src/main/resources/assets/gregtech/lang/en_us.lang

```diff
@@ -2887,6 +2887,7 @@ gregtech.recipe.amperage=Amperage: %,d
 gregtech.recipe.not_consumed=Not consumed in process
 gregtech.recipe.chance=Chance: %s%% +%s%%/tier
 gregtech.recipe.blast_furnace_temperature=Temperature: %,dK (%s)
+gregtech.recipe.explosive=Explosive: %s
```

Comment:
If we are setting the property as hidden, do we need this translation?

Reply:
The property can technically have its hidden state removed, so I figured it would be better to be safe than have an untranslated string show up in JEI.